### PR TITLE
Update README.md with new make targets

### PR DESCRIPTION
**Pull request by adshmh**
Update path_run target with config notes
Update README.md TOC

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -50,30 +70,32 @@ test_e2e_morse_relay: ## Run an E2E Morse relay test
...
-		echo "#############################################################################################################"; \
-		echo "### Created ./config/.config.yaml                                                                         ###"; \
-		echo "### README: Please update the the following in .config.yaml: 'url', 'relay_signing_key', & 'signed_aats'. ###"; \
-		echo "#############################################################################################################"; \
+	@if [ ! -f ./bin/config/.config.yaml ]; then \
+		mkdir -p bin/config; \
+		cp ./cmd/.config.morse_example.yaml ./bin/config/.config.yaml; \
+		echo "#######################################################################################################"; \
+		echo "### Created ./bin/config/.config.yaml                                                               ###"; \
+		echo "### README: Please update the the following in .config.yaml: gateway_private_key & gateway_address. ###"; \
```
Comment (commoddity) [inline Makefile:93]: This should refer to the morse-specific variables: `relay_signing_key`, `signed_aats`, etc.

File: README.md [hunk 2]
```diff
@@ -231,93 +230,112 @@ services:
...
+## 7. Running Localnet
 
 You can use path configuration under `/local` to spin up a local development environment using `Kind` + `Tilt`.
 
 Make sure to review [Tiltfile](https://github.com/buildwithgrove/path/tree/main/Tiltfile) and [values file](https://github.com/buildwithgrove/path/tree/main/local/path/config/path-values.yaml) to make sure they have your desired configuration.
 
-### 8.1 Spinning up / Tearing down Localnet
+### 7.1. Spinning up / Tearing down Localnet
 
 Localnet can be spin up/tear down using the following targets:
```
Comment (commoddity) [inline README.md:333]:
```suggestion
Localnet can be spun up/torn down using the following targets:
```

**Review by commoddity — APPROVED** [review]
LGTM, just a couple nits.

File: Makefile [hunk 3]
```diff
@@ -21,12 +21,32 @@ help: ## Prints all the targets in all the Makefiles
...
+		echo "### ./bin/config/.config.yaml does not exist, use ONE the following to initialize it: ###"; \
+		echo "A. make copy_shannon_config                                                           ###"; \
+		echo "B. make copy_morse_config                                                             ###"; \
+		echo "#########################################################################################"; \
+		exit; \
+	fi; \
+	(cd bin; ./path)
+
+.PHONY: path_up
+path_up: config_shannon_localnet ## Run the PATH gateway and all related dependencies
```
Comment (commoddity) [inline Makefile:37]: Shouldn't this also include the targets for `make localnet_up`, eg.

```make
.PHONY: localnet_up
localnet_up: dev_up config_path_secrets ## Brings up local Tilt development environment (using kind cluster)
	@tilt up

.PHONY: localnet_down
localnet_down: dev_down ## Tears down local Tilt development environment (using kind cluster)
```
https://github.com/buildwithgrove/path/blob/b7a8774704dad447083e3a1017b4a9b219167f1f/Makefile#L206C1-L211C94

**Review by commoddity — CHANGES_REQUESTED** [review]
In addition to my last review, just wanted to confirm if `path_up` and `path_down` should be using the targets for the localnet kind cluster as well?

**Review by Olshansk — APPROVED** [review]
I pushed a bunch of changes to this PR.

@commoddity Please review it and approve it it looks good to you

Note that we're going to keep iterating on it in (your) subsequent PRs regardless.

**Review by commoddity — APPROVED** [review]
LGTM

**Pull request merged**